### PR TITLE
@Hide abt public classes

### PR DESCRIPTION
**Pull request by rlazo**
A/B Testing doesn't have user visible classes, so they should be  annotated with javadoc's @hide.

### CODE REVIEW

File: firebase-abt/src/main/java/com/google/firebase/abt/component/package-info.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/**
+ * @hide
```
Comment (vkryachko) [inline firebase-abt/src/main/java/com/google/firebase/abt/component/package-info.java]: I think the whole comment needs to be on one line, let's see if google-java-format fails on this.

Comment (rlazo) [reply firebase-abt/src/main/java/com/google/firebase/abt/component/package-info.java]: PTAL

Comment (vkryachko) [reply firebase-abt/src/main/java/com/google/firebase/abt/component/package-info.java]: there is a merge conflict in both files, pls double check

Comment (rlazo) [reply firebase-abt/src/main/java/com/google/firebase/abt/component/package-info.java]: fixed

**Pull request merged**